### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by BillDietrich** [canonical/ubuntu-desktop-installer#47]
Will the new installer be usable by sight-impaired users ?  Just raising the issue, since articles online say that Flutter doesn't produce HTML that adheres to accessibility standards.  I'm not impaired myself, and I don't know what kind of "reader" a person could use to handle doing an install.  How do sight-impaired people do an Ubuntu install today ?  Use a char-mode installer ?

**Comment by GiovanniSM20** [comment]
I don't know what you think we could use to solve this problem but we could use something like gnome's [orca.](https://help.gnome.org/users/orca/stable/introduction.html.en) Actually we have some distros using this for solve this problem.

**Comment by robert-ancell** [comment]
The installer won't be using the web renderer on Flutter - instead it draws much the same way as any other toolkit (e.g. GTK, Qt). It exposes the widgets that it uses over AT-SPI, so should have a similar level of accessibility as any other desktop app.

**Comment by BillDietrich** [comment]
Thanks.  How does a sight-impaired person do this in an installer ?  They're booted into a single application.  They have no opportunity to run a screen-reader app.

**Comment by robert-ancell** [comment]
I believe the a11y menu is available during the installer.

**Comment by madsrh** [comment]
@BillDietrich did Robert answer your question sufficient for this issue to be closed?

**Comment by BillDietrich** [comment]
Okay, thanks.
